config.js

```diff
@@ -14,34 +15,35 @@ module.exports = {
   },
   useMocks: useMocks,
   dateTimeFormat: 'DD-MM-YYYY, hh:mma',
-  port: process.env.PORT || 8080,
+  port: port,
   session: {
-    secret: process.env.SESSION_SECRET || 'howdoesyourgardengrow',
-    ttl: process.env.SESSION_TTL || 1800, /* 30 mins timeout */
-    name: process.env.SESSION_KEY || 'rotm.sid'
+    secret: process.env.SESSION_SECRET,
+    ttl: process.env.SESSION_TTL, /* 30 mins timeout */
+    name: process.env.SESSION_KEY
   },
   redis: {
-    port: process.env.REDIS_PORT || '6379',
-    host: process.env.REDIS_HOST || '127.0.0.1',
+    port: process.env.REDIS_PORT,
+    host: process.env.REDIS_HOST,
     password: process.env.REDIS_PASSWORD
   },
   email: {
-    from: process.env.FROM_ADDRESS || '',
-    replyTo: process.env.REPLY_TO || '',
-    region: process.env.EMAIL_REGION || '',
-    caseworker: process.env.CASEWORKER_EMAIL ||  'sas-hof-test@digital.homeoffice.gov.uk',
+    from: process.env.FROM_ADDRESS,
+    replyTo: process.env.REPLY_TO,
+    region: process.env.EMAIL_REGION,
+    transport: process.env.EMAIL_TRANSPORT,
+    caseworker: process.env.CASEWORKER_EMAIL,
     notifyApiKey: process.env.NOTIFY_KEY,
-    notifyTemplate: process.env.NOTIFY_TEMPLATE || '5734dfd3-ea54-4b86-8914-0458d8895559'
+    notifyTemplate: process.env.NOTIFY_TEMPLATE
   },
   hosts: {
-    acceptanceTests: process.env.ACCEPTANCE_HOST_NAME || `http://localhost:${process.env.PORT || 8080}`
+    acceptanceTests: process.env.ACCEPTANCE_HOST_NAME || `http://localhost:${port}`
   },
   upload: {
     maxFileSize: '100mb',
     // if mocks set use file service served up by app otherwise use filevault's port 3000
     hostname: !useMocks && process.env.FILE_VAULT_URL ?
       process.env.FILE_VAULT_URL :
-      `http://localhost:${useMocks ? (process.env.PORT || 8080) : 3000}/file`
+      `http://localhost:${useMocks ? port : 3000}/file`
```

Comment:
if you remove 8080 then you'll need to remove 3000 as that is setting the value

Reply:
I think 3000 is the default port for file-vault set in the [file-vault-deployment.yml](https://github.com/UKHomeOffice/rotm/blob/baf85c6d997d86524c2db9043364dd042ea7bf27/kube/file-vault/file-vault-deployment.yml#L67-L68) 🤔